utils/like_matcher.hh

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
I wonder if there is a way to make Github respect our `.gitorder`.

Reply:
Perhaps not, but you can review the files in [any order you wish](https://github.com/dear-github/dear-github/issues/195#issuecomment-247743919).

Reply:
Review-by file doesn't work for PRs that have more than one commit touching a single file.
BTW resolved comments are hard to find.

Sorry to litter this review with remarks about the PR experience. Maybe we need to collect questions/answers and general feedback in some centralized place.